glamorous/webpack.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  module: {
+    loaders: [
+      {
+        test: /\.js$/,
+        exclude: /(node_modules|bower_components)/,
```

Comment:
I imagine you copy pasted this config. Mind simplifying it a bit by removing the Bower thing. Nobody needs to see that anymore 🙃